src/theme.ts

```diff
@@ -3,5 +3,6 @@ import { theme } from '@terraware/web-components';
 // This import is needed to get the types for customized Palette, unfortunately this is the easiest fix for now.
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import TerrawareThemeOptions from '@terraware/web-components/style-dictionary-dist/TerrawareTheme';
+import TerrawareThemeOptions2 from '@terraware/web-components/style-dictionary-dist2/TerrawareTheme2';
```

Comment:
a million thanks 🙏 for keeping this in mind

Reply:
I actually wasn't sure about this, as I could not see either `TerrawareThemeOptions` or `TerrawareThemeOptions2` exported from those files, but I figured if it compiled successfully then it must be happening where I could not see.

Reply:
Oh it's just importing the default-export from those files, the naming here was intentional (my comments hopefully adds some context).. the main intention of this import is to get the theme options defined in memory - this is purely for typescript to stop complaining. if there's a better way to do this, we should.. maybe something for later.

Reply:
Ohh, gotcha 🎉